src/org/usfirst/frc/team5590/robot/subsystems/Grabber.java

```diff
@@ -0,0 +1,81 @@
+package org.usfirst.frc.team5590.robot.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.command.Subsystem;
+//**MUST IMPORT FOR TALONPORT**
+
+/*
+ * 
+ */
+public class Grabber extends Subsystem {
+
+	private static final int SOLENOID_IN = 2;
```

Comment:
These should be put in `org.usfirst.frc.team5590.robot.RobotMap`. Name them similarly to the Drivetrain ports, but calls them `GR_....` instead. Be descriptive. Also, we will have two talon speed controllers, 1 for each side